tools/clang/test/HLSLFileCheck/hlsl/semantics/legacy/obsolete_semantics_ps_3.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
```

Comment:
FileCheck is still going to need the `-input=stderr` flag in order to catch the warning text. Otherwise it just gets the stdout which doesn't include the warning text.

As a complete aside in case you ever need it, I've occasionally started making a change to allow `-input=stdall` that takes in both stderr and stdout, but every time I've eventually decided I don't really need it just yet. The day will come though!

Reply:
Ah .. sorry for bothering you. I took some time to set up the Windows SDK and WDK. As far as I tested it, now it works. Thank you for your guidance!